### DISCUSSION TITLE
Send Amps to InfluxDB?

### DISCUSSION THREAD

**Discussion by Robert-D** [jertel/vuegraf#39] [Q&A] [answered]
Is it possible to send amps to InfluxDB as well as watts? (By the way this is awesome and thank you so much for this docker!)

**Comment by jertel** [comment, marked as answer]
Amps can be derived from watts via division by voltage, so having it in the DB would incur a double storage cost for no real gain that i know of. Interested in hearing your thoughts on this, though.

**Reply by Robert-D** [reply]
Are volts sent as a metric to influx?

I was just thinking this would be helpful for me to graph amps to watch the trend for my home load. I recently installed a 30amp backfeed generator breaker and feel like the amp draw will trip this breaker way before my generators rated watt capacity is reached.

**Reply by jertel** [reply]
No, the voltages aren't stored in the DB. However, in Grafana you can perform a math function and divide the watts by a static value of 240v, and optionally trigger an alarm, should the resulting value exceed the threshold. 

But you really don't need amps for this. Since your threshold is 30a, and since a 30a breaker would typically be used on a 240v circuit then you're watt threshold is 30*240, or 7200w. So rather than deal with amps conversion you can monitor the wattage directly and see how close it's getting to 7200w.
